packages/ui/Toggleable/tests/Toggleable-specs.js

```diff
@@ -290,7 +290,7 @@ describe('Toggleable', () => {
 				<Component defaultSelected />
 			);
 
-			subject.find(DivComponent).prop('onToggle')();
+			subject.find(DivComponent).invoke('onToggle')();
```

Comment:
`invoke` is a newer method that is required when calling a method that produces a state change via hooks. We need to update CLI to use a more recent enzyme and react adapter for this feature.